diff --git a/node_modules/@sveltejs/kit/src/exports/vite/build/build_service_worker.js b/node_modules/@sveltejs/kit/src/exports/vite/build/build_service_worker.js
index fcdfbb5..11ab35b 100644
--- a/node_modules/@sveltejs/kit/src/exports/vite/build/build_service_worker.js
+++ b/node_modules/@sveltejs/kit/src/exports/vite/build/build_service_worker.js
@@ -79,6 +79,9 @@ export async function build_service_worker(
 			outDir: `${output_dir}/client`,
 			emptyOutDir: false
 		},
+		define: {
+			'process.env.NODE_ENV': '"production"',
+		},
 		configFile: false,
 		resolve: {
 			alias: {
